src/components/__tests__/Toggle-test.tsx

```diff
@@ -63,4 +63,10 @@ describe("Toggle", () => {
     expect(onToggle.args[1][0]).to.be.false;
     isOn(false);
   });
+
+  it("should have an aria-label", () => {
+    let slider = wrapper.find("button");
+
+    expect(slider.prop("aria-label")).to.equal("Toggle button");
+  });
```

Comment:
If we're doing this, then let's also check for aria-labels in the tests for the other components that use them (I know some of them already have, but I don't think all of them do)...

Reply:
Sounds good.